quantum/quantum.c

```diff
@@ -627,14 +634,21 @@ bool process_record_quantum(keyrecord_t *record) {
       }
       else {
         #ifdef DISABLE_SPACE_CADET_ROLLOVER
-          if (get_mods() & MOD_BIT(KC_RSFT)) {
+          if (get_mods() & MOD_BIT(RSPC_MOD)) {
```

Comment:
Have you tested this with DISABLE_SPACE_CADET_ROLLOVER on?

Reply:
And yes as well :smile:

Reply:
The question was repeated accidentally, not to emphasize it ;-).